lib/Form/Block/Text.php

```diff
@@ -41,14 +41,27 @@ public function render(array $block, array $formSettings): string {
     if (in_array($block['id'], ['email', 'last_name', 'first_name'], true)) {
       $automationId = 'data-automation-id="form_' . $block['id'] . '" ';
     }
+    $name = $this->rendererHelper->getFieldName($block);
 
     $html = '';
+    if (
+      isset($block['params']['label_within'])
+      && $block['params']['label_within']
+      && isset($block['styles']['font_color'])
+    ) {
+      $html .= '<style>'
+        . 'input[name="data[' . $name . ']"]::placeholder{'
+        . 'color:' . $block['styles']['font_color'] . ';'
+        . 'opacity: 1;'
+        . '}'
+        . '</style>';
+    }
```

Comment:
This code repeats itself twice. I think we could add it to `BlockStylesRenderer` as a method. What do you think?